suit_generator/suit/security.py

```diff
@@ -162,22 +170,61 @@ class SuitDigest(SuitUnion):
     _metadata = Metadata(children=[SuitDigestRaw, SuitDigestExt])
 
 
-class SuitcoseSignAlg(SuitEnum):
+class SuitcoseAlg(SuitEnum):
     """Representation of SUIT COSE sign algorithm."""
 
-    _metadata = Metadata(children=[cose_alg_es_256, cose_alg_es_384, cose_alg_es_521, cose_alg_eddsa])
+    _metadata = Metadata(children=[cose_alg_es_256, cose_alg_es_384, cose_alg_es_521, cose_alg_eddsa,
+                                   cose_alg_aes_gcm_128, cose_alg_aes_gcm_192, cose_alg_aes_gcm_256,
+                                   cose_alg_a256kw, cose_alg_a192kw, cose_alg_a128kw,
+                                   ])
+
+class SuitcoseKeyId(SuitUnion): # TODO - should we allow the text as well to match the example from the spec?
```

Comment:
I'd skip it and do not define the text as an option.

Reply:
Deleted TODO, as discussed